src/sdk/pynni/nni/medianstop_assessor/test.py

```diff
@@ -31,11 +31,11 @@ def test():
     #       [1,1,1,1,1,1,1,1,1,1],
     #       [1,1,1,1,1,1,1,1,1,1]]
 
-    assessor = MedianstopAssessor(FLAGS.start_step, FLAGS.optimize_mode)
-    for i in range(4):
+    assessor = MedianstopAssessor(FLAGS.optimize_mode, FLAGS.start_step)
+    for i in range(len(lcs)):
         #lc = []
         to_complete = True
-        for k in range(10):
+        for k in range(FLAGS.start_step):
```

Comment:
`FLAGS.start_step` -> `len(lcs[0])`

Reply:
`FLAGS.start_step` is internal used by `MedianstopAssessor`

Reply:
@QuanluZhang  Fixed